cli/cmd/install.go

```diff
@@ -216,6 +216,10 @@ A full list of configurable values can be found at https://www.github.com/linker
 
 func newCmdInstall() *cobra.Command {
 	values, err := l5dcharts.NewValues()
+	if err != nil {
+		fmt.Fprint(os.Stderr, err.Error())
```

Comment:
should this have a newline?
```suggestion
		fmt.Fprintln(os.Stderr, err.Error())
```